### PR TITLE
REF: pseudosingle is now a Device

### LINKED ISSUE #344: PseudoSingle should be a Device
With a setpoint that can be recorded

Ref https://github.com/NSLS-II/bluesky/issues/496

### PR DESCRIPTION
**Pull request by klauer**
Allows `setpoint` and `readback` to be read as normal `Device` `Components`

Closes #344

### CODE REVIEW

**Comment by klauer** [comment]
Failing due to error-handling coverage - will update

File: ophyd/pseudopos.py [hunk 1]
```diff
@@ -466,15 +490,30 @@ def connected(self):
...
-
-        super().stop()
+                exc_list.append((attr, ex))
+                logger.error('Device %s (%s) stop failed', attr, dev,
+                             exc_info=ex)
+
+        if exc_list:
+            exc_info = '\n'.join('{} raised {!r}'.format(attr, ex)
+                                 for attr, ex in exc_list)
+            raise ExceptionBundle('{} exception(s) were raised during stop: \n'
```
Comment (danielballan) [inline ophyd/pseudopos.py]: I'm really glad we adopted this pattern.

**Comment by danielballan** [comment]
Seems like a good idea to me!

**Comment by arkilic** [comment]
👏

**Pull request merged**